src/constants.ts

```diff
@@ -31,6 +32,7 @@ export const IPC_CHANNELS = {
   SHOW_DIRECTORY_PICKER: 'show-directory-picker',
   INSTALL_COMFYUI: 'install-comfyui',
   SHOW_CONTEXT_MENU: 'show-context-menu',
+  GET_OS_PLATFORM: 'get-os-platform',
```

Comment:
Why do we need the `OS_PLATFORM` in electron API?

Reply:
Per specification the menu item should show only if the application is ran on Win32 platform so I need to way to determine that in frontend.

Reply:
Can we have the getOS API added in a separate PR?